pkg/api/v1/status/serverless_private_endpoint.go

```diff
@@ -27,48 +25,16 @@ type ServerlessPrivateEndpoint struct {
 	PrivateLinkServiceResourceID string `json:"privateLinkServiceResourceId,omitempty"`
 }
 
-func SPEFromAtlas(in mongodbatlas.ServerlessPrivateEndpointConnection) ServerlessPrivateEndpoint {
-	return ServerlessPrivateEndpoint{
-		ID:                           in.ID,
-		CloudProviderEndpointID:      in.CloudProviderEndpointID,
-		Name:                         in.Comment,
-		EndpointServiceName:          in.EndpointServiceName,
-		ErrorMessage:                 in.ErrorMessage,
-		Status:                       in.Status,
-		ProviderName:                 in.ProviderName,
-		PrivateEndpointIPAddress:     in.PrivateEndpointIPAddress,
-		PrivateLinkServiceResourceID: in.PrivateLinkServiceResourceID,
-	}
-}
-
-func FailedToCreateSPE(comment, message string) ServerlessPrivateEndpoint {
-	return ServerlessPrivateEndpoint{
-		ErrorMessage: message,
-		Name:         comment,
-		Status:       StatusFailed,
-	}
-}
-
-func FailedDuplicationSPE(name, cloudProviderEndpointID, privateEndpointIPAddress string) ServerlessPrivateEndpoint {
-	return ServerlessPrivateEndpoint{
-		CloudProviderEndpointID:  cloudProviderEndpointID,
-		PrivateEndpointIPAddress: privateEndpointIPAddress,
-		ErrorMessage:             fmt.Sprintf("The SPE with same name exists: %s. Please use unique names", name),
-		Name:                     name,
-		Status:                   StatusFailed,
-	}
-}
-
-func FailedToConnectSPE(pe mongodbatlas.ServerlessPrivateEndpointConnection, message string) ServerlessPrivateEndpoint {
+func SPEStatusFromAtlas(in *admin.ServerlessTenantEndpoint) ServerlessPrivateEndpoint {
 	return ServerlessPrivateEndpoint{
-		ID:                           pe.ID,
-		CloudProviderEndpointID:      pe.CloudProviderEndpointID,
-		Name:                         pe.Comment,
-		EndpointServiceName:          pe.EndpointServiceName,
-		ErrorMessage:                 message,
-		Status:                       StatusFailed,
-		ProviderName:                 pe.ProviderName,
-		PrivateEndpointIPAddress:     pe.PrivateEndpointIPAddress,
-		PrivateLinkServiceResourceID: pe.PrivateLinkServiceResourceID,
+		ID:                           in.GetId(),
+		Name:                         in.GetComment(),
```

Comment:
This seems counter-intuitive. Please make a godoc comment why `GetComment()` is associated with the name.